<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Depositar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(122, 82, 208);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>560</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>560</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="buttonBackDep">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 18pt &quot;Verdana&quot; ;
	background-color: rgb(255, 255, 255);
	border: rgb(255, 255, 255);
	color: rgb(122, 82, 208);
}

QPushButton:pressed{
	color: rgb(0, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>130</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Depositar</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="inpValDep">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>220</y>
         <width>280</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	font: 8pt &quot;Verdana&quot; ;
	border: 2px solid rgb(130, 130,130);
	border-radius: 20px;
	padding: 15px;
	background-color: rgb(255, 255, 255);
	color: rgb(122, 82, 208);
}

QLineEdit:hover{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(122, 82, 208);
}

QLineEdit:focus{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(122, 82, 208);
}</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>50000</number>
       </property>
       <property name="placeholderText">
        <string>VALOR</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonDep">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>310</y>
         <width>280</width>
         <height>50</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 8pt &quot;Verdana&quot; ;
	border: 2px solid rgb(122, 82, 208);
	border-radius: 5px;
	padding: 15px;
	color: rgb(122, 82, 208);
	background-color: rgb(255, 255, 255);
}

QPushButton:pressed{
	background-color: rgb(122, 82, 208);
	color: rgb(0, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>CONTINUAR</string>
       </property>
      </widget>
      <widget class="QLabel" name="statusTrans">
       <property name="geometry">
        <rect>
         <x>115</x>
         <y>280</y>
         <width>251</width>
         <height>20</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="labelStatusDep">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>180</y>
         <width>261</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(0, 230, 230)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
